addons/web_editor/static/src/js/wysiwyg/widgets/link.js

```diff
@@ -252,7 +252,7 @@ const Link = Widget.extend({
         const $linkUrlInput = this.$('#o_link_dialog_url_input');
         const value = $linkUrlInput.val();
         const isLink = value.indexOf('@') < 0;
-        this.$('input[name="is_new_window"]').closest('.form-group').toggleClass('d-none', !isLink);
+        this._getIsNewWindowFormRow().toggleClass('d-none', !isLink);
```

Comment:
I don't understand what this commit has to do with the rest. Also, I believe this bug was fixed and merged two weeks ago (https://www.odoo.com/web#id=2789222&menu_id=4720&cids=1&action=333&active_id=1695&model=project.task&view_type=form) or am I missing something?

Reply:
It has nothing to do with it indeed, this is the reason why I have put it in a distinct commit. I am just trying to make sense of what is being done in the link-related classes.
The task you reference fixes another aspect of the problem, but does not fix the visibility of the option inside the link tools form.

Reply:
Yes the task does correct the visibility of the option inside the link tools form. You didn't see it because it does so in _master_ and not 15.0, as it actually was an improvement and not a fix per se. The option in fact used to work properly, it just happens that the feature it was providing was not useful in the context of mailings, for which the mail client will generally force all links to open in new windows anyway.

Reply:
I did verify on runbot's master before posting my message: if I type an e-mail address in the URL of link tools, the option remains visible.
![image](https://user-images.githubusercontent.com/71644421/208403607-74b71d4c-dee7-40a4-aff3-ea9e737abdf0.png)

Reply:
Alright, that is kinda weird. We'll investigate this as a followup to the task in master.

Reply:
Hello there! I get this on runbot's master, on mass mailing:
![image](https://user-images.githubusercontent.com/65516503/208408053-db818927-e382-4fe7-ae3c-3674a34626ea.png)

Reply:
My screenshot was taken on website, where the option is supposed to be visible or invisible - which is what my commit is addressing. In mass mailing it is supposed to always be hidden, and it works fine in master.

Reply:
Did anyone even read and/or tried to reproduced BSO clear step to reproduce in the commit message before commenting?

Reply:
> Did anyone even read and/or tried to reproduced BSO clear step to reproduce in the commit message before commenting?

I didn't because I assumed it was related to the initial purpose of this PR which specifically only pertained to `mass_mailing`. Sorry for not noticing earlier that this fix was in fact completely unrelated to the original issue and is in fact targeting a completely different bug in a completely different context. 😉